routers/user/profile.go

```diff
@@ -184,6 +184,7 @@ func Profile(ctx *context.Context) {
 				ctx.ServerError("GetRepositories", err)
 				return
 			}
+			models.RepositoryList(repos).LoadAttributes()
```

Comment:
Same here, shouldn't this make it slower?

Reply:
No.

Reply:
Can you explain? I totally don't get this why this should speed things up. It's doing more queries (useless, also! since we know the owner must always be of the profile's user.)